coder-observability/runbooks/coderd.md

```diff
@@ -82,4 +82,54 @@ Please contact your Coder sales contact, or visit https://coder.com/contact/sale
 Prebuilds only become eligible to be claimed by users once the workspace's agent is a) running and b) all of its startup
 scripts have completed.
 
-If a prebuilt workspace is not eligible, view its agent logs to diagnose the problem.
\ No newline at end of file
+If a prebuilt workspace is not eligible, view its agent logs to diagnose the problem.
+
+## CoderdUnprovisionedPrebuiltWorkspaces
+
+The number of running prebuilt workspaces is lower than the desired instances. This could be for several reasons,
+ordered by likehood:
+
+### Experiment/License
+
+The prebuilds feature is currently gated behind an experiment *and* a premium license.
```

Comment:
Consider something like:

```
Prebuilds require a premium license. Ensure that your premium license is inplace.

Additionally, your version of Coder might require an experiment. Consult the documentation for your version of Coder to determine whether you need to enable `CODER_EXPERIMENTS=workspace-prebuilds`
```

Because:
* This gives an additional hint to the operator so that they can determine whether or not it is in fact the experiment. On later versions where it might no longer be an experiment, they might enable/disable it to no effect and be uncertain of the outcome. This will make troubleshooting harder
* Its more resilient against drift by referring to a version specific source of truth.

Reply:
I'd rather we went back and updated this doc once the feature is no longer an experiment.
I've added a reminder for myself.